app/assets/javascripts/admin/bulk_product_update.js.coffee

```diff
@@ -212,10 +212,10 @@ angular.module("ofn.admin").controller "AdminProductEditCtrl", ($scope, $timeout
       ).error (data, status) ->
         if status == 400 && data.errors? && data.errors.length > 0
           errors = error + "\n" for error in data.errors
-          alert "Saving failed with the following error(s):\n" + errors
-          $scope.displayFailure "Save failed due to invalid data"
+          alert t("products_update_error") + errors
```

Comment:
Are u sure?
What I have :
 ).error (data, status) ->
        if status == 400 && data.errors? && data.errors.length > 0
          errors = error + "\n" for error in data.errors
          alert t("products_update_error") + errors
          $scope.displayFailure t("products_update_error")
        else
          $scope.displayFailure t("products_update_error_data") + status
It sounds good..